packages/accounts/plugingen/phases/plugin-generator/account-method-gen.ts

```diff
@@ -53,7 +53,7 @@ export const AccountMethodGenPhase: Phase = async (input) => {
   input.content.push(dedent`
     accountMethods: (${
       hasViewFunction ? "account" : "_account"
-    }: IMSCA<any, any>) => ({ ${accountFunctions.join(",\n\n")} })
+    }: IMSCA<any, any, any>) => ({ ${accountFunctions.join(",\n\n")} })
```

Comment:
yea I remember needing the `any, any` originally because of the builder. I couldn't find a way to avoid it, but could be a good follow up investigation for the future